src/ResourceGraph/ResourceGraph/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 [assembly: CLSCompliant(false)]
 [assembly: Guid("cdfd4e1b-bb5f-439b-8843-e6127cd9b8ad")]
 
-[assembly: AssemblyVersion("0.8.0")]
-[assembly: AssemblyFileVersion("0.8.0")]
+[assembly: AssemblyVersion("0.9.0")]
```

Comment:
Please revert these changes bc we have version controller do such version bump things.